src/app/layout/Page/Page.jsx

```diff
@@ -17,6 +17,15 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+/**
+ * Represents a Customised Page.
+ * @param {object} appbar - The Title of the app bar.
+ * @param {object} children - The content of the page.
+ * @param {object} title - The Title of the page.
```

Comment:
can we be more specify here, as in string, bool, and such.

Reply:
@mat10tng Sure. Have updated and pushed the changes. Please review.😃